src/components/application_manager/include/application_manager/resumption/resumption_data_db.h

```diff
@@ -34,8 +34,13 @@
 #define SRC_COMPONENTS_APPLICATION_MANAGER_INCLUDE_APPLICATION_MANAGER_RESUMPTION_RESUMPTION_DATA_DB_H_
 #include "application_manager/resumption/resumption_data.h"
 
+#ifdef __QNX__
+#include "utils/qdb_wrapper/sql_database.h"
```

Comment:
@BSolonenko can we try to resolve this  cross platform issue in cmakes?

Reply:
@AKalinich-Luxoft Fixed in b2b51c312fc57007af9a6626c0e88bfe2f0b594e